src/services/locationService.js

```diff
@@ -0,0 +1,60 @@
+import * as env from '../../env';
+import * as Utils from '../utils/utils';
+import {
+  RESPONSE_FAILURE,
+  RESPONSE_OFFICE_PLACE,
+  RESPONSE_SUCCESS_RETRY_PLACE
+} from '../modules/office/OfficeState';
+
+const CLIENT_ID = env.SERVICE_CLIENT_ID;
+const CLIENT_SECRET = env.SERVICE_CLIENT_SECRET;
+const CATEGORY_ID = '4d4b7105d754a06374d81259'; //CatergoryId for 'Food'
+const RADIUS = '500'; //In meters
+
+var venues = [];
```

Comment:
Why is this mutable var needed? Can't you declare `venues` locally in the function scope? If it's needed in more places then it needs to live in the redux store otherwise this `locationService` has some unwanted state.